torchaudio/transforms.py

```diff
@@ -649,17 +648,7 @@ def forward(self, waveform: Tensor) -> Tensor:
             Tensor: Output signal of dimension (..., time).
         """
         if self.resampling_method == 'sinc_interpolation':
```

Comment:
note: we'll leave the discussion around migrating the `resampling_method` parameter to a follow-up PR